rpcclient/infrastructure.go

```diff
@@ -768,11 +768,14 @@ func (c *Client) handleSendPostMessage(jReq *jsonRequest) {
 	}
 	url := protocol + "://" + c.config.Host
 
-	var err error
-	var backoff time.Duration
-	var httpResponse *http.Response
+	var (
+		err          error
+		backoff      time.Duration
+		httpResponse *http.Response
+	)
+
 	tries := 10
-	for i := 0; tries == 0 || i < tries; i++ {
```

Comment:
Also seems to be inserting a bug fix within this formatting fix commit? 

Unclear what the purpose of that is, given that `tries` is never decremented...

Reply:
I think this may have been left when someone was debugging retries and they set `tries` to zero to induce infinite retries.